src/ui/atoms/button/Button.tsx

```diff
@@ -42,22 +54,34 @@ export const Button: React.FC<TButtonProps> = ({
   backgroundColor = 'primary',
   disabled = false,
   label,
+  leftIcon,
+  rightIcon,
+  icon,
   onClick
 }: TButtonProps): JSX.Element => (
   <button
     className={classNames('okp4-button-main', {
       [`variant-${variant}`]: true,
       [`size-${size}`]: true,
       [`background-color-${backgroundColor}`]: true,
-      disabled: !!disabled
+      disabled,
+      icon: variant === 'icon'
```

Comment:
Why add a `icon` className as the `variant-icon` className already exists? 🧐

Reply:
Good question 😪  Another miss